core/src/smartcontracts/isi/account.rs

```diff
@@ -59,7 +59,12 @@ pub mod isi {
             wsv.modify_account(&account_id, |account| {
                 account
                     .remove_asset(&asset_id)
-                    .map(|asset| AccountEvent::Asset(AssetEvent::Removed(asset.id().clone())))
+                    .map(|asset| {
+                        AccountEvent::Asset(AssetEvent::Removed(AssetChangedBy {
+                            asset_id: asset.id().clone(),
+                            by: asset.value().clone(),
```

Comment:
`by` is ambiguous. I don't know off the top of my head what it means and need to dig into the source code to find out.